pkg/logger/logger.go

```diff
@@ -188,6 +188,9 @@ func NewZapLogger(level zapcore.Level) (*zap.Logger, error) {
 		},
 		DisableCaller: false,
 	}
+	if cfg.Encoding == "" {
```

Comment:
these lines can be removed as the default is already set via the flag definition itself:
```
flags.StringP(logFormatFlagName, "z", "console", "Set the logging format, e.g. console or json ")
```